docs/tutorial/workers.md

```diff
@@ -50,10 +54,7 @@ graph TD
     worker --> |creates| flow_run_infra
 ```
 
-!!! note "Security Note"
-    Prefect provides execution through its hybrid model, which allows you to deploy workflows that run in the environments best suited to their execution while allowing you to keep your code and data completely private. 
-    There is no ingress required. 
-    For more information [read more about our hybrid model](https://www.prefect.io/security/overview/#overview).
+<sup>Notice above that the worker is in charge of provisioning the _flow run infra_. In context of this tutorial, the worker will spin up an ephemeral docker image to host each flow run. Different [worker types](/concepts/work-pools/#worker-types) can create different types of flow run infrastructure.</sup>
```

Comment:
This is what I say out loud to people when I walk them through the diagram and it always helps get an aha out of them.